### ISSUE TITLE
Create Google docstring without types if the function uses type hints

### ISSUE THREAD

**Issue by MicaelJarniac** [NilsJPWerner/autoDocstring#210]
```py
def function_with_pep484_type_annotations(param1: int, param2: str) -> bool:
    """Example function with PEP 484 type annotations.

    Args:
        param1: The first parameter.
        param2: The second parameter.

    Returns:
        The return value. True for success, False otherwise.

    """
```
> https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

> [...] The description should include required type(s) if the code does not contain a corresponding type annotation. [...]
> https://google.github.io/styleguide/pyguide.html#doc-function-args

**Comment by ebursztein** [comment]
I concur, that would be great. In particular as you type function using standard python annotations, the types are on the arguments, not in the doc strings. In general I would favor, not having types, with the possibility to include as option if possible. https://docs.python.org/3/library/typing.html

**Comment by evanharwin** [comment]
Just commenting to say that I came to this repo to find more details on precisely this. Would be great to see this supported.

**Comment by NilsJPWerner** [comment]
Hey sorry for the radio silence. I am going to find some time to support this in january.

**Comment by dgasmith** [comment]
Same issue on our end, this would be a fantastic feature to have!

**Comment by NilsJPWerner** [comment]
Fixed in #209 and #221
